kubernetes/windows/main.ps1

```diff
@@ -607,12 +640,18 @@ if (![string]::IsNullOrEmpty($requiresCertBootstrap) -and `
 $isAADMSIAuth = [System.Environment]::GetEnvironmentVariable("USING_AAD_MSI_AUTH")
 if (![string]::IsNullOrEmpty($isAADMSIAuth) -and $isAADMSIAuth.ToLower() -eq 'true') {
     Write-Host "skipping agent onboarding via cert since AAD MSI Auth configured"
+
+    #start Windows AMA 
+    Start-Job -ScriptBlock { Start-Process -NoNewWindow -FilePath "C:\opt\genevamonitoringagent\genevamonitoringagent\Monitoring\Agent\MonAgentLauncher.exe" -ArgumentList @("-useenv")}
```

Comment:
If MonAgentLauncher.exe fails to start or crash, where the logs will be to debug why its failed to start or crash?